controllers/reconcile.go

```diff
@@ -853,6 +853,94 @@ func setupFedModules(feEnv *crd.FrontendEnvironment, frontendList *crd.FrontendL
 	return nil
 }
 
+func adjustSearchEntry(searchEntry *crd.SearchEntry, frontend crd.Frontend) crd.SearchEntry {
+	newSearchEntry := crd.SearchEntry{
+		// make the id environment and frontend specific to reduce duplicate ids across Frontend resources
+		ID:          fmt.Sprintf("%s-%s-%s", frontend.Name, frontend.Spec.EnvName, searchEntry.ID),
```

Comment:
@Hyperkid123 I'm assuming we chose this over a true UUID as we need this to be predictable/not create a new search entry every update correct?

Reply:
Yes, we need to know if there are duplicates. It has to be deterministic. The format does not matter but we gotta know if there is the same entry twice.